### PR TITLE
fixed default snapshot and statesync dirs

### PR DESCRIPTION
**Pull request by brennanjl**
Fixes the default directors for snapshot and statesync not being in the root directory.

### CODE REVIEW

**Comment by jchappelow** [comment]
In some default toml files we have:

```toml
# The directory where the snapshots are stored. Can be absolute or relative to the kwild root directory
snapshot_dir = "snapshots"
```

which gives me the following error when running the binary from somewhere the process does not have permission to write to:

```
Error: failed to sanitize config paths: failed to expand snapshot directory "snapshots": failed to get absolute path of file: snapshots due to error: stat .: permission denied
```

With the change in this PR, there's no sensible default for the toml, unless it's a full absolute path?

**Comment by brennanjl** [comment]
The default could just be `~/.kwild/snapshots` right? Or does that not work on linux

**Comment by jchappelow** [comment]
Let's just update the default config files that still specify "snapshots".  I'd prefer to have this be relative to root dir though.

> The default could just be `~/.kwild/snapshots` right? Or does that not work on linux

Same on linux, it's just problematic because what evaluates the "~" depends on where it is set.  If it's in config.toml, then it's the process' owner.  If it' on the cli flags, then it's the shell of the person running the command. These are often different.  When defining say a systemd service, it's going to launch as a user specified by the systemd .service file and a working directory that's generally irrelevant to the process.

**Comment by brennanjl** [comment]
So we just need to remove the value from the default config?

**Comment by charithabandi** [comment]
Oh shoot. I messed this up. Created same branch name as yours and the fix got pushed here. I get why jon recommends doing it from a fork

**Comment by brennanjl** [comment]
> Oh shoot. I messed this up. Created same branch name as yours and the fix got pushed here. I get why jon recommends doing it from a fork

Lol no worries, I was going to close this in favor of what you put up.

File: build/package/docker/kwild.dockerfile [hunk 1]
```diff
@@ -19,10 +19,10 @@ RUN GOWORK=off GIT_VERSION=$version GIT_COMMIT=$git_commit BUILD_TIME=$build_tim
 RUN GOWORK=off GIT_VERSION=$version GIT_COMMIT=$git_commit BUILD_TIME=$build_time CGO_ENABLED=0 TARGET="/app/dist" GO_RACEFLAG=$go_race ./scripts/build/binary kwil-cli
 RUN chmod +x /app/dist/kwild /app/dist/kwil-admin /app/dist/kwil-cli
 
-FROM alpine:3.19
+FROM ubuntu:22.04
```
Comment (jchappelow) [inline build/package/docker/kwild.dockerfile:22]: 24.04?  No reason to start with the old release, right?

**Review by jchappelow — APPROVED** [review]

**Pull request merged**